cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -430,7 +430,6 @@ std::shared_ptr<velox::Config> WholeStageResultIterator::createConnectorConfig()
       veloxCfg_->get<bool>(kCaseSensitive, false) == false ? "true" : "false";
   configs[velox::connector::hive::HiveConfig::kArrowBridgeTimestampUnit] = "6";
   configs[velox::connector::hive::HiveConfig::kMaxPartitionsPerWritersSession] = "400";
-  configs[velox::connector::hive::HiveConfig::kPartitionPathAsLowerCaseSession] = "false";
```

Comment:
@JkSelf This config was added into #3690 but the build complains that it is not available. Can you please check if it is safe to remove?

Reply:
@Surbhi-Vijay This configuration is used to handle the differences in partition path names between Velox and Spark. It has already been added to the `HiveConfig` class in the `2023_12_27` branch of the OAP Velox repository [here](https://github.com/oap-project/velox/commit/8e0e247e770511d5b8df1114c78720716175f80b#diff-c56b08dbdccf3a0637ed66aa44683a26a58eee3bd2136c4754ee5959c1cdf647R113). Could you please try using either the `2023_12_27 `or `2023_12_28 `branch?

Reply:
Using the newer OAP Velox branches fixes the issue of config not found.